### ISSUE TITLE
sir its not working

### ISSUE THREAD

**Issue by LedArts** [smokeandlights/Intensity-Wave#1]
hello sir,
i am using ardiuno uno 
i try this example but not working
please help me
i only change the pin 0 to pin 6

**Comment by smokeandlights** [comment]
Yeah, I am not very good at programming and never got it to work. Good luck!

Sent from TypeApp<http://www.typeapp.com/r?b=9447>
On Apr 10, 2017, at 5:13 PM, LedArts <notifications@github.com<mailto:notifications@github.com>> wrote:

hello sir,
i am using ardiuno uno
i try this example but not working
please help me
i only change the pin 0 to pin 6
